### ISSUE TITLE
onSelect event not working correctly.

### ISSUE THREAD

**Issue by JamesHarrisonSage** [blazeworx/flagstrap#17]
You can even see an issue on your demo page. 
http://blazeworx.github.io/flagstrap/

The example with onSelect works for the first selection if you choose US. After that it returns null for all other selections. 

We have had to use a workaround using jquery. 

$("#countrySelector").find("select").change(function() {

```
   var selectedCountry = $(this).children('[selected]').val();
```

});

**Comment by caroso1222** [comment]
Yep. Same here. It seems that the mistake is in setting the selected attribute to "selected" and then trying to get the select value with `select.val()`. A better workaround would be to use the plugin onSelect callback but append this line before using the `value`:

```javascript
$('.flagstrap').flagStrap({
   onSelect: function (value, element) {
     value = $(element).children("option[selected=selected]").val(); //This is the correct value
     //Now you can use it
     console.log("This is the selected country code: " + value);
   }
});
```

**Comment by yakar** [comment]
Its work excellent, thank you @caroso1222

**Comment by peterb-2795** [comment]
Nice country select plugin @blazeworx +  neat fix @caroso1222

**Comment by blazeworx** [comment]
Closed by #20
